shell/platform/android/io/flutter/embedding/engine/android/FlutterFragment.java

```diff
@@ -308,6 +330,67 @@ protected String getDartEntrypointFunctionName() {
   public void onPostResume() {
     Log.d(TAG, "onPostResume()");
     flutterEngine.getLifecycleChannel().appIsResumed();
+
+    // TODO(mattcarroll): find a better way to handle the update of UI overlays than calling through
+    //                    to platformPlugin. We're implicitly entangling the Window, Activity, Fragment,
+    //                    and engine all with this one call.
+    platformPlugin.onPostResume();
+
+    // TODO(mattcarroll): consider a more abstract way to invoke this behavior. It is very strange for
+    //                    a Fragment to have a seemingly random View responsibility, but this is what
+    //                    existed in the original embedding and I don't have a good alternative yet.
+    flutterView.updateAccessibilityFeatures();
```

Comment:
Again, the engine could be handling this it should know if there's a view available, and be able to manage letting the view know it needs to update this.

Reply:
(I originally misunderstood and though the view actually was setting something important on itself here - that appears to not be the case, and this really should just be a method on FlutterEngine AFAICT)

Reply:
Accessibility, at best, would be a responsibility of the `AccessibilityBridge` rather than the `FlutterEngine`.

Reply:
The bridge doesn't know anything about the renderer though, does it?

Reply:
We seem to be going down a number of parallel rabbit holes here.  I was trying to demonstrate iwth these TODOs that the answers aren't obvious and that's why I'm not solving them in this PR.  I would prefer that we not attempt to solve 3 complicated problems via code review.  Let's chat in person if you'd like to work on these solutions before their own respective PRs go up.  At the moment I'm just bringing FlutterFragment up to date with the corresponding existing behavior in the old embedding.